cpp/velox/jni/VeloxJniWrapper.cc

```diff
@@ -111,6 +111,8 @@ Java_io_glutenproject_vectorized_PlanEvaluatorJniWrapper_nativeValidateWithFailu
 
   // A query context used for function validation.
   velox::core::QueryCtx queryCtx;
+  std::unordered_map<std::string, std::string> configs{{velox::core::QueryConfig::kSparkPartitionId, "0"}};
+  queryCtx.testingOverrideConfigUnsafe(std::move(configs));
```

Comment:
Seems `testingOverrideConfigUnsafe` should be used in test code only.

Reply:
Updated, thanks.